data/json/items/ammo/10mm.json

```diff
@@ -2,7 +2,7 @@
   {
     "id": "10mm_fmj",
     "type": "AMMO",
-    "name": { "str": "10mm Auto FMJ" },
+    "name": { "str_sp": "10mm Auto FMJ" },
```

Comment:
Where do these strings appear in-game? If I were to suggest "X rounds of 10mm Auto FMJ," and similarly for the other ammo types, would it break the screen too badly?

Reply:
The context is mixed. It can be talking about what ammunition a gun takes, in which case "round of"/"rounds of" is just repeating the obvious. Or it can be talking about something in a container, in which case singular vs plural does make a difference. (And there are probably other cases that I'm not thinking of - neither @BrettDong nor @Qrox were quite sure of all the different places they could be used; see the earlier PR I referenced.)

Reply:
Oof, yeah. And I am *loathe* to suggest adding another field (and the relevant C++ support) to handle separating those cases.

Reply:
What BrettDong was wanting to do was convert as much to str_sp as possible to alleviate burden on the translators (even worse than excess entries were things like "9mm NATO, reloaded" becoming "9mm NATO, reloadeds" and some poor person being asked to translate that...).

Reply:
Ahh. OK, if the primary goal is to help the translators, then I think it's OK to accept a little bit of "maybe this isn't *super* grammatical" in service to that.

Reply:
Of course, the description below appears to be contradictory with the "count" field, although that the "stack_size" is identical may be an explanation (I _think_ that means you can work with one at a time, but I'm not sure).

Reply:
Probably that last one; there's a lot of duct tape and scuff marks in the game's code at the moment. :)

Reply:
According to `doc/JSON_INFO.md`,

```c++
"count" : 25,         // Number of rounds that spawn together
"stack_size" : 50,    // (Optional) How many rounds are in the above-defined volume. If omitted, is the same as 'count'
```

So a charge of ammo is a round of ammo. The description of BBs is a bit confusing here.